tldextract/tldextract.py

```diff
@@ -204,6 +218,19 @@ def __init__(self, cache_file=CACHE_FILE, suffix_list_urls=PUBLIC_SUFFIX_LIST_UR
         self.extra_suffixes = extra_suffixes
         self._extractor = None
 
+        self.cache_fetch_timeout = None
+        if CACHE_TIMEOUT is not None:
```

Comment:
This seems to take the opposite precedence order from what CACHE_FILE takes: with the `cache_file` argument you can override the environment variable using the keyword argument. On the other hand, if there exists a valid value for `cache_fetch_timeout` available from the environment then the API does not allow overriding it with the keyword argument.

Is there an advantage to preferring this argument precedence here?

Reply:
Can I blame coding after bed time? I'm gonna blame coding after bed time.

Good catch as always @sseg, thanks!